### PR TITLE
SEO-175665 - Angular doc changes

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @MallikaRK ,

Worked on page with redirect for Angular.

Regards,
Sharly A.

### CODE REVIEW

File: ej2-angular/toast/config.md [hunk 1]
```diff
@@ -95,5 +95,5 @@ Both width and height property allows setting pixels/numbers/percentage. The num
 
 ## See Also
 
-* [Prevent duplicate toasts](./how-to/prevent-duplicate-toast-display/)
+* [Prevent duplicate toasts](./how-to/prevent-duplicate-toast-display)
 * [Customize the progress bar](./how-to/customize-progress-bar-theme-and-sizing/)
```
Comment (Yvone-Atieno) [inline ej2-angular/toast/config.md:99]: Check the above link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-angular/toast/config.md (hunk 1 above)
Comment (Sharly-Atieno) [reply ej2-angular/toast/config.md]: Resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**